src/anomalib/data/noise.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
```

Comment:
Can you add license and docstring at the top? Also, how about naming this to `noise_type.py` similar to `task_type.py`

Reply:
fixed.